examples/functions-google-gmail/README.md

```diff
@@ -31,7 +31,7 @@
     When developing locally, you include environment variables to your `.env.development`. Read more about how Gatsby handles `.env` files and environment variables in the [Gatsby Docs](https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/)
 
     ```shell
-    cd google-auth
+    cd google-gmail
```

Comment:
But that's not the name of the directory? If we want to be super correct it has to be `cd examples/functions-google-gmail`